From fc3db84daaf5612dac1a9c09b1ec188a3ec2c3d3 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 27 Mar 2012 17:11:00 +0200
Subject: [PATCH] units: get rid of var-run.mount and var-lock.mount

Since a number of distribitions don't need this compat glue anymore drop
it from systemd upstream. Distributions which still haven't converted
to /run can steal these unit files from the git history if they need to.
(cherry picked from commit 55d029addf7d97b15faacea597a4ff65542aaf0e)

Conflicts:

	Makefile.am
---
 Makefile.am          |   14 --------------
 units/var-lock.mount |   19 -------------------
 units/var-run.mount  |   19 -------------------
 3 files changed, 0 insertions(+), 52 deletions(-)
 delete mode 100644 units/var-lock.mount
 delete mode 100644 units/var-run.mount

diff --git a/Makefile.am b/Makefile.am
index d563cfd..63bb817 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -295,7 +295,6 @@ dist_systemunit_DATA = \
 	units/sys-kernel-config.mount \
 	units/sys-kernel-debug.mount \
 	units/sys-fs-fuse-connections.mount \
-	units/var-run.mount \
 	units/media.mount \
 	units/remount-rootfs.service \
 	units/printer.target \
@@ -308,11 +307,6 @@ dist_systemunit_DATA = \
 	units/systemd-ask-password-console.path \
 	units/syslog.target
 
-if HAVE_SYSV_COMPAT
-dist_systemunit_DATA += \
-	units/var-lock.mount
-endif
-
 nodist_systemunit_DATA = \
 	units/getty@.service \
 	units/serial-getty@.service \
@@ -2341,12 +2335,10 @@ systemd-install-data-hook:
 		rm -f systemd-remount-api-vfs.service \
 			fsck-root.service \
 			remount-rootfs.service \
-			var-run.mount \
 			media.mount && \
 		$(LN_S) ../systemd-remount-api-vfs.service systemd-remount-api-vfs.service && \
 		$(LN_S) ../fsck-root.service fsck-root.service && \
 		$(LN_S) ../remount-rootfs.service remount-rootfs.service && \
-		$(LN_S) ../var-run.mount var-run.mount && \
 		$(LN_S) ../media.mount media.mount )
 	( cd $(DESTDIR)$(userunitdir) && \
 		rm -f shutdown.target sockets.target bluetooth.target printer.target sound.target && \
@@ -2506,12 +2498,6 @@ if TARGET_MAGEIA
 		$(LN_S) $(systemunitdir)/display-manager.service display-manager.service )
 endif
 
-if HAVE_SYSV_COMPAT
-	( cd $(DESTDIR)$(systemunitdir)/local-fs.target.wants && \
-		rm -f var-lock.mount && \
-		$(LN_S) ../var-lock.mount var-lock.mount )
-endif
-
 install-exec-hook: $(INSTALL_EXEC_HOOKS)
 
 uninstall-hook: $(UNINSTALL_EXEC_HOOKS)
diff --git a/units/var-lock.mount b/units/var-lock.mount
deleted file mode 100644
index 07277ad..0000000
--- a/units/var-lock.mount
+++ /dev/null
@@ -1,19 +0,0 @@
-#  This file is part of systemd.
-#
-#  systemd is free software; you can redistribute it and/or modify it
-#  under the terms of the GNU General Public License as published by
-#  the Free Software Foundation; either version 2 of the License, or
-#  (at your option) any later version.
-
-[Unit]
-Description=Lock Directory
-Before=local-fs.target
-# skip mounting if the directory does not exist or is a symlink
-ConditionPathIsDirectory=/var/lock
-ConditionPathIsSymbolicLink=!/var/lock
-
-[Mount]
-What=/run/lock
-Where=/var/lock
-Type=bind
-Options=bind
diff --git a/units/var-run.mount b/units/var-run.mount
deleted file mode 100644
index ab4da42..0000000
--- a/units/var-run.mount
+++ /dev/null
@@ -1,19 +0,0 @@
-#  This file is part of systemd.
-#
-#  systemd is free software; you can redistribute it and/or modify it
-#  under the terms of the GNU General Public License as published by
-#  the Free Software Foundation; either version 2 of the License, or
-#  (at your option) any later version.
-
-[Unit]
-Description=Runtime Directory
-Before=local-fs.target
-# skip mounting if the directory does not exist or is a symlink
-ConditionPathIsDirectory=/var/run
-ConditionPathIsSymbolicLink=!/var/run
-
-[Mount]
-What=/run
-Where=/var/run
-Type=bind
-Options=bind
